README.md

```diff
@@ -56,6 +56,19 @@ If you run it this way, you'll want to put the configuration file in a system wi
 
 Currently the exporter just writes all relevant output, including errors, to stdout. If you have any problems with it starting or it remaining started, be sure to look through the systemd journal for any errors with something like the following: `journalctl -u ntopng-exporter --since="5m ago"`
 
+
+## docker
+Another option is to use docker to deploy the service. You can build the docker image yourself, or use one published on https://hub.docker.com.
+To build the image, make sure that you have at least docker 17.05 installed, and build using the command `docker build .` in the checked out repository.
+
+To run the docker image, a volume containing the `ntopng-exporter.yaml` configuration file named `/config` must be mounted and the port needs to be
+exposed, either with a dynamic port allocation witht the `--publish-all` option or with a specific port mapping with `--publish`. An example
```

Comment:
```suggestion
exposed, either with a dynamic port allocation with the `--publish-all` option or with a specific port mapping with `--publish`. An example
```

Small spelling correction, otherwise looks good!

Reply:
Good catch!